jest.config.js

```diff
@@ -13,7 +13,4 @@ module.exports = {
     "json",
     "node"
   ],
-  moduleNameMapper: {
-    "(.*)@openstax/ts-utils(.*)": "$1@openstax/ts-utils/$2.js"
```

Comment:
I thought this was doing the same thing as manually adding the extensions to the imports, maybe not?

Reply:
It was fine in the spec files but then in the regular imports that needed the extension outside of jest it started duplicating the extension for me.